src/NUnitFramework/tests/Internal/Filters/NotFilterTests.cs

```diff
@@ -67,7 +67,7 @@ public void ExplicitMatchTest()
         {
             var filter = new NotFilter(new CategoryFilter("Dummy"));
 
-            Assert.False(filter.IsExplicitMatch(_topLevelSuite));
+            Assert.True(filter.IsExplicitMatch(_topLevelSuite));
```

Comment:
I defined it as being the same as Match.  It seemed that was the correct thing to do.